### PR TITLE
Added a basic achievement component

### PR DESCRIPTION
**Pull request by Tanish2000**
Screenshot (88)
Screenshot (89)

### CODE REVIEW

File: src/modules/resume/achievements/variants/basic/render.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+const Basic = (
+  {
+    theme,
+    headlineText,
+    achievements
+  }
+) => {
+  const styles = useStyles()
+  function render_list() {
```
Comment (WebDevRaj) [inline src/modules/resume/achievements/variants/basic/render.js:23]: Use camel case for function names

File: src/modules/resume/achievements/variants/basic/render.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }}>
+      <h1 style={styles.headlineText}>{headlineText.value}</h1>
+      <hr />
+      {renderList()}
+    </div >
+
+  );
+};
+
+export default Basic;
```
Comment (RisingGeek) [inline src/modules/resume/achievements/variants/basic/render.js:49]: Add a newline to every file. Use prettier to format code.

File: src/modules/resume/achievements/variants/basic/contributors.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const contributors = [
+  {
+    name: 'Tanish Singh Chouhan',
+    email: 'chouhantanish@gmail.com',
+    url: 'https://github.com/Tanish2000'
+  }
+];
+
+export default contributors;
```
Comment (RisingGeek) [inline src/modules/resume/achievements/variants/basic/contributors.js:9]: Please add newline at the end of each file

Comment (RisingGeek) [reply src/modules/resume/achievements/variants/basic/contributors.js:9]: Not that important.

Comment (Tanish2000) [reply src/modules/resume/achievements/variants/basic/contributors.js:9]: Okay

**Comment by Tanish2000** [comment]
When it will be merged?

**Comment by RisingGeek** [comment]
It should be merged when 3 reviewers approve. I think only @ManojSatishkumar has write access

**Comment by ManojSatishkumar** [comment]
Check now. 
I gave merge access

**Pull request merged**